### ISSUE TITLE
Problems after upgrade wandb server to the latest version

### ISSUE THREAD

**Issue by JiarunLiu** [wandb/server#108]
Hi, I was upgrade my wandb server to the latest version (docker image id: `06e5925be5dc` ==> `b46f78b7ffa8`) yesterday. But there are something wrong after the upgrade. 

First, the web interface could not rendering my projects:
<img width="967" alt="image" src="https://user-images.githubusercontent.com/42335666/218402113-ade3bdfc-1e4a-43b9-a26d-0613704756df.png">

The other problems is I see some error message at the beginning & end of my training:
```
wandb: ERROR Error uploading "wandb-metadata.json": CommError, <Response [403]>
wandb: ERROR Error uploading "code/main.py": CommError, <Response [403]>

...

wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: ERROR Error uploading "config.yaml": CommError, <Response [403]>
wandb: ERROR Error uploading "requirements.txt": CommError, <Response [403]>
wandb: ERROR Error uploading "conda-environment.yaml": CommError, <Response [403]>
wandb: ERROR Error uploading "output.log": CommError, <Response [403]>
wandb: ERROR Error uploading "wandb-summary.json": CommError, <Response [403]>
```

Here are some messages filtered by "ERROR" at `my_run/logs/debug-internal.log` 
```shell
2023-02-13 15:35:29,423 ERROR   Thread-17 :3721785 [internal_api.py:upload_file():1924] upload_file exception http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/code/main.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073529Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=0af5f14fe29884c8c442f92daaff3a049eb4977f2d3cd9c1423ed4499524ae14: 403 Client Error: Forbidden for url: http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/code/main.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073529Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=0af5f14fe29884c8c442f92daaff3a049eb4977f2d3cd9c1423ed4499524ae14
2023-02-13 15:35:29,423 ERROR   Thread-16 :3721785 [internal_api.py:upload_file():1924] upload_file exception http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073529Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=a9c014fa796cbaaed0f4f9547185d64aedcad43b0b2f56a752bafd18f7edf7a5: 403 Client Error: Forbidden for url: http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073529Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=a9c014fa796cbaaed0f4f9547185d64aedcad43b0b2f56a752bafd18f7edf7a5
2023-02-13 15:35:29,423 ERROR   Thread-17 :3721785 [internal_api.py:upload_file():1926] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '15055', 'Authorization': 'Basic dXNlcjpsb2NhbC0yYjQ1MDJhYmEzNzNmMWQzZTZkNzA2ZGVkMzZmZDMwZTU4ODdlMWVj'}
2023-02-13 15:35:29,423 ERROR   Thread-16 :3721785 [internal_api.py:upload_file():1926] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5585', 'Authorization': 'Basic dXNlcjpsb2NhbC0yYjQ1MDJhYmEzNzNmMWQzZTZkNzA2ZGVkMzZmZDMwZTU4ODdlMWVj'}
2023-02-13 15:35:29,424 ERROR   Thread-17 :3721785 [internal_api.py:upload_file():1928] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Key>jiarunliu/stDLNN/3o5oqdf5/code/main.py</Key><BucketName>local-files</BucketName><Resource>/local-files/jiarunliu/stDLNN/3o5oqdf5/code/main.py</Resource><RequestId>174352135BF4EF4C</RequestId><HostId>d617ec31-ea6d-44f3-90f2-89cb4a611522</HostId></Error>'
2023-02-13 15:35:29,424 ERROR   Thread-16 :3721785 [internal_api.py:upload_file():1928] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Key>jiarunliu/stDLNN/3o5oqdf5/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/jiarunliu/stDLNN/3o5oqdf5/wandb-metadata.json</Resource><RequestId>174352135BF4EF4C</RequestId><HostId>d617ec31-ea6d-44f3-90f2-89cb4a611522</HostId></Error>'
2023-02-13 15:35:29,429 ERROR   Thread-17 :3721785 [upload_job.py:push():146] Failed to upload file: /tmp/tmpr5bwdzl2wandb/1tbhto28-code/main.py
2023-02-13 15:35:29,431 ERROR   Thread-16 :3721785 [upload_job.py:push():146] Failed to upload file: /tmp/tmpr5bwdzl2wandb/e5dvmova-wandb-metadata.json
2023-02-13 15:37:16,434 ERROR   Thread-18 :3721785 [internal_api.py:upload_file():1924] upload_file exception http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=691eced824995a6c87c347c64e1f0398151bd88d3c6d61c244e0404ddc878658: 403 Client Error: Forbidden for url: http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=691eced824995a6c87c347c64e1f0398151bd88d3c6d61c244e0404ddc878658
2023-02-13 15:37:16,434 ERROR   Thread-18 :3721785 [internal_api.py:upload_file():1926] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1044', 'Authorization': 'Basic dXNlcjpsb2NhbC0yYjQ1MDJhYmEzNzNmMWQzZTZkNzA2ZGVkMzZmZDMwZTU4ODdlMWVj'}
2023-02-13 15:37:16,438 ERROR   Thread-18 :3721785 [internal_api.py:upload_file():1928] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Key>jiarunliu/stDLNN/3o5oqdf5/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/jiarunliu/stDLNN/3o5oqdf5/config.yaml</Resource><RequestId>1743522C46B46025</RequestId><HostId>d617ec31-ea6d-44f3-90f2-89cb4a611522</HostId></Error>'
2023-02-13 15:37:16,447 ERROR   Thread-20 :3721785 [internal_api.py:upload_file():1924] upload_file exception http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=1ea82a51c65d4491d4869ac08f675075e9697446c126d1afaf0a825fe87ae542: 403 Client Error: Forbidden for url: http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=1ea82a51c65d4491d4869ac08f675075e9697446c126d1afaf0a825fe87ae542
2023-02-13 15:37:16,450 ERROR   Thread-18 :3721785 [upload_job.py:push():146] Failed to upload file: /home/fgldlb/Documents/stDLNN/wandb/run-20230213_153523-3o5oqdf5/files/config.yaml
2023-02-13 15:37:16,450 ERROR   Thread-20 :3721785 [internal_api.py:upload_file():1926] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3087', 'Authorization': 'Basic dXNlcjpsb2NhbC0yYjQ1MDJhYmEzNzNmMWQzZTZkNzA2ZGVkMzZmZDMwZTU4ODdlMWVj'}
2023-02-13 15:37:16,451 ERROR   Thread-21 :3721785 [internal_api.py:upload_file():1924] upload_file exception http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=3fb9e3f2022d36980ff660ae303676349f53d1beb3eba8a899b38ce95d63bf4b: 403 Client Error: Forbidden for url: http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=3fb9e3f2022d36980ff660ae303676349f53d1beb3eba8a899b38ce95d63bf4b
2023-02-13 15:37:16,453 ERROR   Thread-19 :3721785 [internal_api.py:upload_file():1924] upload_file exception http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/wandb-summary.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=dc6be5065e6438b2a2d772a118615cbf3f73bb9626164a1dfc42880cc45c2736: 403 Client Error: Forbidden for url: http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/wandb-summary.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=dc6be5065e6438b2a2d772a118615cbf3f73bb9626164a1dfc42880cc45c2736
2023-02-13 15:37:16,455 ERROR   Thread-22 :3721785 [internal_api.py:upload_file():1924] upload_file exception http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=1c91ae2023cdbfed33796501e2ae9f351cc9303d2e279939d52f4a15fedd3155: 403 Client Error: Forbidden for url: http://localhost:8080/local-files/jiarunliu/stDLNN/3o5oqdf5/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=8eYcMlh0NyJX6qfZk7BL0g%3D%3D%2F20230213%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20230213T073716Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=jiarunliu&X-Amz-Signature=1c91ae2023cdbfed33796501e2ae9f351cc9303d2e279939d52f4a15fedd3155
2023-02-13 15:37:16,455 ERROR   Thread-20 :3721785 [internal_api.py:upload_file():1928] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Key>jiarunliu/stDLNN/3o5oqdf5/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/jiarunliu/stDLNN/3o5oqdf5/requirements.txt</Resource><RequestId>1743522C477D17FC</RequestId><HostId>d617ec31-ea6d-44f3-90f2-89cb4a611522</HostId></Error>'
2023-02-13 15:37:16,456 ERROR   Thread-21 :3721785 [internal_api.py:upload_file():1926] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '7990', 'Authorization': 'Basic dXNlcjpsb2NhbC0yYjQ1MDJhYmEzNzNmMWQzZTZkNzA2ZGVkMzZmZDMwZTU4ODdlMWVj'}
2023-02-13 15:37:16,456 ERROR   Thread-19 :3721785 [internal_api.py:upload_file():1926] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '28', 'Authorization': 'Basic dXNlcjpsb2NhbC0yYjQ1MDJhYmEzNzNmMWQzZTZkNzA2ZGVkMzZmZDMwZTU4ODdlMWVj'}
2023-02-13 15:37:16,457 ERROR   Thread-22 :3721785 [internal_api.py:upload_file():1926] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1444', 'Authorization': 'Basic dXNlcjpsb2NhbC0yYjQ1MDJhYmEzNzNmMWQzZTZkNzA2ZGVkMzZmZDMwZTU4ODdlMWVj'}
2023-02-13 15:37:16,457 ERROR   Thread-21 :3721785 [internal_api.py:upload_file():1928] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Key>jiarunliu/stDLNN/3o5oqdf5/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/jiarunliu/stDLNN/3o5oqdf5/conda-environment.yaml</Resource><RequestId>1743522C47BF560D</RequestId><HostId>d617ec31-ea6d-44f3-90f2-89cb4a611522</HostId></Error>'
2023-02-13 15:37:16,459 ERROR   Thread-19 :3721785 [internal_api.py:upload_file():1928] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Key>jiarunliu/stDLNN/3o5oqdf5/wandb-summary.json</Key><BucketName>local-files</BucketName><Resource>/local-files/jiarunliu/stDLNN/3o5oqdf5/wandb-summary.json</Resource><RequestId>1743522C4778CBF3</RequestId><HostId>d617ec31-ea6d-44f3-90f2-89cb4a611522</HostId></Error>'
2023-02-13 15:37:16,460 ERROR   Thread-20 :3721785 [upload_job.py:push():146] Failed to upload file: /home/fgldlb/Documents/stDLNN/wandb/run-20230213_153523-3o5oqdf5/files/requirements.txt
2023-02-13 15:37:16,460 ERROR   Thread-22 :3721785 [internal_api.py:upload_file():1928] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>AccessDenied</Code><Message>Access Denied.</Message><Key>jiarunliu/stDLNN/3o5oqdf5/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/jiarunliu/stDLNN/3o5oqdf5/output.log</Resource><RequestId>1743522C47CC4512</RequestId><HostId>d617ec31-ea6d-44f3-90f2-89cb4a611522</HostId></Error>'
2023-02-13 15:37:16,470 ERROR   Thread-19 :3721785 [upload_job.py:push():146] Failed to upload file: /home/fgldlb/Documents/stDLNN/wandb/run-20230213_153523-3o5oqdf5/files/wandb-summary.json
2023-02-13 15:37:16,472 ERROR   Thread-21 :3721785 [upload_job.py:push():146] Failed to upload file: /home/fgldlb/Documents/stDLNN/wandb/run-20230213_153523-3o5oqdf5/files/conda-environment.yaml
2023-02-13 15:37:16,477 ERROR   Thread-22 :3721785 [upload_job.py:push():146] Failed to upload file: /home/fgldlb/Documents/stDLNN/wandb/run-20230213_153523-3o5oqdf5/files/output.log

```

**Comment by nate-wandb** [comment]
Hi @JiarunLiu, it looks like the server is not able to write anything to the attached bucket. I imagine you see the same error if you try to log an Artifact correct? 

Can you run `wandb verify` in the CLI (not in the Docker container but just on a machine logged into your local server) and let me know the output? 

Thank you,
Nate

**Comment by JiarunLiu** [comment]
邮件已收到！

**Comment by nate-wandb** [comment]
Thank you @JiarunLiu! It looks the issue is with the SQS configuration for the bucket. You can ignore the failed https test since you use http. Can you go through the [steps here](https://docs.wandb.ai/guides/self-hosted/setup/configuration#file-storage) for your external bucket and make sure this is setup correctly? 

Thank you,
Nate

**Comment by JiarunLiu** [comment]
Thank you @nate-wandb! I didn't find the applied operation for me in [this document](https://docs.wandb.ai/guides/self-hosted/setup/configuration#file-storage) cause I'm not using any external file storage (like Azure or Google Cloud?). I upgrade my wandb server to the newly released version `0.28.0` without setting local variable `LOCAL_RESTORE=true`. Now everything is back to normal. Thank you again!
